packages/react-core/src/components/Select/Select.tsx

```diff
@@ -353,36 +353,44 @@ export class Select extends React.Component<SelectProps & OUIAProps, SelectState
           return React.cloneElement(group, {
```

Comment:
This looks like more than release notes. Do you want these Select changes reviewed too?

Reply:
oops. thought I stashed those!